Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Staging/SetPoints/Subsequences/FailsafeCondition.mo

```diff
@@ -0,0 +1,150 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.BoilerPlant.Staging.SetPoints.Subsequences;
+block FailsafeCondition
+  "Failsafe condition used in staging up and down"
+
+  parameter Real TDif(
+    final unit="K",
+    final displayUnit="K",
+    final quantity="TemperatureDifference") = 10
+    "Required temperature difference between setpoint and measured temperature
+    for failsafe condition";
+
+  parameter Real TDifHys(
+    final unit="K",
+    final displayUnit="K",
+    final quantity="TemperatureDifference") = 1
+    "Temperature deadband for hysteresis loop"
+    annotation (Dialog(tab="Advanced"));
+
+  parameter Real delEna(
+    final unit="s",
+    final displayUnit="s",
+    final quantity="Time") = 900
+    "Enable delay";
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uStaChaProEnd
+    "Signal indicating end of stage change process"
+    annotation (Placement(transformation(extent={{-160,-70},{-120,-30}}),
+      iconTransformation(extent={{-140,-70},{-100,-30}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TSupSet(
+    final unit="K",
+    final displayUnit="K",
+    final quantity="ThermodynamicTemperature")
+    "Hot water supply temperature setpoint"
+    annotation (Placement(transformation(extent={{-160,30},{-120,70}}),
+      iconTransformation(extent={{-140,30},{-100,70}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TSup(
+    final unit="K",
+    final displayUnit="K",
+    final quantity="ThermodynamicTemperature")
+    "Measured hot water supply temperature"
+    annotation (Placement(transformation(extent={{-160,-20},{-120,20}}),
+      iconTransformation(extent={{-140,-20},{-100,20}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yFaiCon
+    "Failsafe condition for boiler staging"
+    annotation (Placement(transformation(extent={{120,-20},{160,20}}),
+      iconTransformation(extent={{100,-20},{140,20}})));
+
```

Comment:
1 empty line is enough probably, although I personally don't have preferences on this.